docs/sources/static/set-up/install-agent-binary.md

```diff
@@ -1,32 +1,49 @@
 ---
-title: Install static mode on other systems
+title: Other systems
 weight: 140
 aliases:
 - ../../set-up/install-agent-binary/
 ---
 
-## Install static mode on other systems
+# Install Grafana Agent on other systems
 
-Install Grafana Agent and get it up and running using a binary file.
+Grafana Agent Flow is distributed as plain binaries for various systems:
```

Comment:
Did you mean to say Flow here? This is the static mode install instructions.

Reply:
Copy/paste error, I'll fix it